src/core.jl

```diff
@@ -109,10 +177,10 @@ end
 """
     degree_histogram(g, degfn=degree)
 
-Return a Dict with values representing the number of vertices that have degree
+Return a `Dict` with values representing the number of vertices that have degree
```

Comment:
should we use 3rd person for docs? Like "Returns" ?

Reply:
`Return` is base Julia style